### PR TITLE
PAN-OS - Added the *edl_type*, *location*, and *vsys* arguments to the ***panorama-refresh-edl***

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/28847

## Description
Please see the referenced issue

## Minimum version of Demisto
- [x] 3.5.0

## Does it break backward compatibility?
   - [x] No

Will add a note in the docs soon

### CODE REVIEW

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 1]
```diff
@@ -3596,16 +3596,28 @@ def panorama_delete_edl_command():
...
     }
+    # if refreshing an EDL on the FW
+    if not edl_type and not location and not vsys:
+        edl = panorama_get_edl(edl_name)
+        edl_type = ''.join(edl['type'].keys())
+    # if refreshing an EDL on the Panorama
+    else:
+        if not edl_type or not location or not vsys:
+            raise Exception('To refresh an EDL from the Firewall on Panorama'
+                            ' please use the: edl_type, location an vsys arguments.')
```
Comment (yuvalbenshalom) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:3612]:
```suggestion
                            ' please use the: edl_type, location and vsys arguments.')
```

**Review by yuvalbenshalom — COMMENTED** [review]
maybe cover this with a unit test?

**Comment by bakatzir** [comment]
@yuvalbenshalom added task to TPB and readme.

**Review by yuvalbenshalom — APPROVED** [review]

**Comment by bakatzir** [comment]
fails on non related `PAN-OS Query Logs For Indicators Test`, merging.

**Pull request merged**